### PR TITLE
docs: document TBS config options for standalone apm server

### PR DESCRIPTION
**Pull request by kruskall**
## Motivation/summary

Update apm-server config files with new tbs settings and list them in the config option page.

## Checklist

~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~
~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
- [x] Documentation has been updated

For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## Related issues

Closes https://github.com/elastic/apm-server/issues/10718

### CODE REVIEW

File: apm-server.docker.yml [hunk 1]
```diff
@@ -279,6 +279,18 @@ apm-server:
...
+
+  #sampling.tail:
+    #Set to `true` to enable tail based sampling. Disabled by default.
+    #enabled: false
+
+    #Synchronization interval for multiple APM Servers. Should be in the order of tens of seconds or low minutes.
+    #interval: 1m
+
+    #Criteria used to match a root transaction to a sample rate.
+    #policies: []
```
Comment (marclop) [inline apm-server.docker.yml:292]: Maybe we can add a commented policy? So users have an easier time writing these?

File: apm-server.docker.yml [hunk 2]
```diff
@@ -279,6 +279,18 @@ apm-server:
     # Configure curve types for ECDHE based cipher suites.
     #ssl.curve_types: []
 
+  #---------------------------- APM Server - tail-based sampling ----------------------------
+
+  #sampling.tail:
+    #Set to `true` to enable tail based sampling. Disabled by default.
```
Comment (dmathieu) [inline apm-server.docker.yml:285]: Nit: other doc comments in the file have a space right after the `#`.

File: apm-server.docker.yml (hunk 1 above)
Comment (kruskall) [reply apm-server.docker.yml:292]: I think the example is currently reflecting the default values. 

I think examples should go in the doc which already has an example configuration for TBS. WDYT ?

Comment (marclop) [reply apm-server.docker.yml:292]: Sure, let's leave it like that then

**Pull request merged**